src/components/CardItem.vue

```diff
@@ -93,5 +91,39 @@ button.visual:hover {
   text-align: center;
   margin: 2px;
 }
+.card0::before {
```

Comment:
ah interesting method, would setting a max width of the number of characters in rems not force a linebreak?

Reply:
Wow did not think of that! I'll leave it at this right now because this took so much time already. I'm also not sure if escaped unicode characters and max number of characters work together nicely. I will keep this in mind when dealing with ASCII in the next project.